MIT-LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright 2017 hdf1986
```

Comment:
copy paste this one https://github.com/Wolox/wor-authentication/blob/master/LICENSE.md